nomic/project.py

```diff
@@ -344,7 +346,8 @@ class AtlasProjection:
     Instead instantiate an AtlasProject and use the project.indices or get_map method to retrieve an AtlasProjection.
     '''
 
-    def __init__(self, project, atlas_index_id: str, projection_id: str, name):
+
+    def __init__(self, project : "AtlasProject", atlas_index_id: str, projection_id: str, name):
```

Comment:
Ahh this is how you type it

Reply:
Only in 3.7 and below! Works fine later.